src/styles/base/_tables.scss

```diff
@@ -16,8 +16,13 @@ td {
 	}
 }
 
-tr:last-child td,
-tr:last-child th {
+tbody:last-child > tr:last-child td {
```

Comment:
I suppose I only tested chrome... but if you inspect a table - it actually adds a `tbody` if you haven't specified one